app/routes/quotes/components/Quotes.css

```diff
@@ -216,7 +216,7 @@
   padding: 0;
 }
 
-.commentCount {
+.reactionCount {
```

Comment:
Consider removing this if you remove the DOM element here https://github.com/webkom/lego-webapp/pull/1622/files#diff-f31e3b34ecfb11e6258937d6ff8c84eaL84